### PR TITLE
Irminger deploy csv updates

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Updated sensor SNs based on written configs from cruise (see attached).
GI01SUMO:
80 & 130 m PCO2Ws were swapped
GI03FLMA:
100 m up OSNAP CTDMOH was incorrect SN
GI03FLMB:
100 m, 400 m, 700 m up OSNAP CTDMOHs were incorrect SNs

[GI01SUMO-00008_2021-08-24.pdf](https://github.com/WHOIGit/ooicgsn-asset-management/files/7093734/GI01SUMO-00008_2021-08-24.pdf)
[GI03FLMA-00008_2021-08-24.pdf](https://github.com/WHOIGit/ooicgsn-asset-management/files/7093735/GI03FLMA-00008_2021-08-24.pdf)
[GI03FLMB-00008_2021-08-24.pdf](https://github.com/WHOIGit/ooicgsn-asset-management/files/7093736/GI03FLMB-00008_2021-08-24.pdf)

### CODE REVIEW

File: deployment/GI01SUMO_Deploy.csv [hunk 1]
```diff
@@ -369,8 +369,8 @@ AR60-01,,,,GI01SUMO-RII11-02-FLORDG031,8,1,2021-08-12T17:04:00,,CGMGI-01SUMO-000
 AR60-01,,,,GI01SUMO-RII11-02-FLORDG032,8,1,2021-08-12T17:04:00,,CGMGI-01SUMO-00008,,CGINS-FLORDG-03969,59.946233,-39.57565,,80,2662,,,
 AR60-01,,,,GI01SUMO-RII11-02-FLORDG033,8,1,2021-08-12T17:04:00,,CGMGI-01SUMO-00008,,CGINS-FLORDG-03970,59.946233,-39.57565,,130,2662,,,
 AR60-01,,,,GI01SUMO-RII11-02-PCO2WC051,8,1,2021-08-12T17:04:00,,CGMGI-01SUMO-00008,,CGINS-PCO2WC-C0105,59.946233,-39.57565,,40,2662,,,
-AR60-01,,,,GI01SUMO-RII11-02-PCO2WC052,8,1,2021-08-12T17:04:00,,CGMGI-01SUMO-00008,,CGINS-PCO2WC-C0121,59.946233,-39.57565,,80,2662,,,
-AR60-01,,,,GI01SUMO-RII11-02-PCO2WC053,8,1,2021-08-12T17:04:00,,CGMGI-01SUMO-00008,,CGINS-PCO2WC-C0122,59.946233,-39.57565,,130,2662,,,
+AR60-01,,,,GI01SUMO-RII11-02-PCO2WC052,8,1,2021-08-12T17:04:00,,CGMGI-01SUMO-00008,,CGINS-PCO2WC-C0122,59.946233,-39.57565,,80,2662,,,
```
Comment (reedan88) [inline deployment/GI01SUMO_Deploy.csv:372]: Looks good.

File: deployment/GI01SUMO_Deploy.csv [hunk 2]
```diff
@@ -369,8 +369,8 @@ AR60-01,,,,GI01SUMO-RII11-02-FLORDG031,8,1,2021-08-12T17:04:00,,CGMGI-01SUMO-000
 AR60-01,,,,GI01SUMO-RII11-02-FLORDG032,8,1,2021-08-12T17:04:00,,CGMGI-01SUMO-00008,,CGINS-FLORDG-03969,59.946233,-39.57565,,80,2662,,,
 AR60-01,,,,GI01SUMO-RII11-02-FLORDG033,8,1,2021-08-12T17:04:00,,CGMGI-01SUMO-00008,,CGINS-FLORDG-03970,59.946233,-39.57565,,130,2662,,,
 AR60-01,,,,GI01SUMO-RII11-02-PCO2WC051,8,1,2021-08-12T17:04:00,,CGMGI-01SUMO-00008,,CGINS-PCO2WC-C0105,59.946233,-39.57565,,40,2662,,,
-AR60-01,,,,GI01SUMO-RII11-02-PCO2WC052,8,1,2021-08-12T17:04:00,,CGMGI-01SUMO-00008,,CGINS-PCO2WC-C0121,59.946233,-39.57565,,80,2662,,,
-AR60-01,,,,GI01SUMO-RII11-02-PCO2WC053,8,1,2021-08-12T17:04:00,,CGMGI-01SUMO-00008,,CGINS-PCO2WC-C0122,59.946233,-39.57565,,130,2662,,,
+AR60-01,,,,GI01SUMO-RII11-02-PCO2WC052,8,1,2021-08-12T17:04:00,,CGMGI-01SUMO-00008,,CGINS-PCO2WC-C0122,59.946233,-39.57565,,80,2662,,,
+AR60-01,,,,GI01SUMO-RII11-02-PCO2WC053,8,1,2021-08-12T17:04:00,,CGMGI-01SUMO-00008,,CGINS-PCO2WC-C0121,59.946233,-39.57565,,130,2662,,,
```
Comment (reedan88) [inline deployment/GI01SUMO_Deploy.csv:373]: Looks good

File: deployment/GI03FLMA_Deploy.csv [hunk 3]
```diff
@@ -182,7 +182,7 @@ AR46,,AR60-01,,GI03FLMA-RIS01-03-DOSTAD000,7,1,2020-08-19T14:48:00,2021-08-17T09
 AR46,,AR60-01,,GI03FLMA-RIS01-04-PHSENF000,7,1,2020-08-19T14:48:00,2021-08-17T09:36:00,CGMGI-03FLMA-00007,,CGINS-PHSENF-P0104,59.767117,-39.843617,,30,2700,,,
 AR46,,AR60-01,,GI03FLMA-RIS01-05-FLORTD000,7,1,2020-08-19T14:48:00,2021-08-17T09:36:00,CGMGI-03FLMA-00007,,CGINS-FLORTD-01300,59.767117,-39.843617,,30,2700,,,
 AR60-01,,,,GI03FLMA-RI000-00-CTDMOH000,8,1,2021-08-13T13:22:00,,CGMGI-03FLMA-00008,,CGINS-CTDMOH-10227,59.77095,-39.8836,,1692,2692,,,
-AR60-01,,,,GI03FLMA-RI000-00-CTDMOH100,8,1,2021-08-13T13:22:00,,CGMGI-03FLMA-00008,,CGINS-CTDMOH-13454,59.77095,-39.8836,,2592,2692,,,
+AR60-01,,,,GI03FLMA-RI000-00-CTDMOH100,8,1,2021-08-13T13:22:00,,CGMGI-03FLMA-00008,,CGINS-CTDMOH-12619,59.77095,-39.8836,,2592,2692,,,
```
Comment (reedan88) [inline deployment/GI03FLMA_Deploy.csv:185]: Looks good.

File: deployment/GI03FLMB_Deploy.csv [hunk 4]
```diff
@@ -182,9 +182,9 @@ AR46,,AR60-01,,GI03FLMB-RIS01-03-DOSTAD000,7,1,2020-08-20T15:25:00,2021-08-18T14
 AR46,,AR60-01,,GI03FLMB-RIS01-04-PHSENF000,7,1,2020-08-20T15:25:00,2021-08-18T14:06:00,CGMGI-03FLMB-00007,,CGINS-PHSENF-P0105,59.716333,-39.31365,,30,2828,,,
 AR46,,AR60-01,,GI03FLMB-RIS01-05-FLORTD000,7,1,2020-08-20T15:25:00,2021-08-18T14:06:00,CGMGI-03FLMB-00007,,CGINS-FLORTD-01301,59.716333,-39.31365,,30,2828,,,
 AR60-01,,,,GI03FLMB-RI000-00-CTDMOH000,8,1,2021-08-17T19:16:00,,CGMGI-03FLMB-00008,,CGINS-CTDMOH-12218,59.71748,-39.35165,,1820,2820,,,
-AR60-01,,,,GI03FLMB-RI000-00-CTDMOH100,8,1,2021-08-17T19:16:00,,CGMGI-03FLMB-00008,,CGINS-CTDMOH-12619,59.71748,-39.35165,,2720,2820,,,
-AR60-01,,,,GI03FLMB-RI000-00-CTDMOH400,8,1,2021-08-17T19:16:00,,CGMGI-03FLMB-00008,,CGINS-CTDMOH-12395,59.71748,-39.35165,,2420,2820,,,
-AR60-01,,,,GI03FLMB-RI000-00-CTDMOH700,8,1,2021-08-17T19:16:00,,CGMGI-03FLMB-00008,,CGINS-CTDMOH-12387,59.71748,-39.35165,,2120,2820,,,
+AR60-01,,,,GI03FLMB-RI000-00-CTDMOH100,8,1,2021-08-17T19:16:00,,CGMGI-03FLMB-00008,,CGINS-CTDMOH-12395,59.71748,-39.35165,,2720,2820,,,
```
Comment (reedan88) [inline deployment/GI03FLMB_Deploy.csv:185]: Looks good.

File: deployment/GI03FLMB_Deploy.csv [hunk 5]
```diff
@@ -182,9 +182,9 @@ AR46,,AR60-01,,GI03FLMB-RIS01-03-DOSTAD000,7,1,2020-08-20T15:25:00,2021-08-18T14
...
+AR60-01,,,,GI03FLMB-RI000-00-CTDMOH400,8,1,2021-08-17T19:16:00,,CGMGI-03FLMB-00008,,CGINS-CTDMOH-12387,59.71748,-39.35165,,2420,2820,,,
```
Comment (reedan88) [inline deployment/GI03FLMB_Deploy.csv:186]: Looks good.

File: deployment/GI03FLMB_Deploy.csv [hunk 6]
```diff
@@ -182,9 +182,9 @@ AR46,,AR60-01,,GI03FLMB-RIS01-03-DOSTAD000,7,1,2020-08-20T15:25:00,2021-08-18T14
...
+AR60-01,,,,GI03FLMB-RI000-00-CTDMOH700,8,1,2021-08-17T19:16:00,,CGMGI-03FLMB-00008,,CGINS-CTDMOH-12220,59.71748,-39.35165,,2120,2820,,,
```
Comment (reedan88) [inline deployment/GI03FLMB_Deploy.csv:187]: Looks good.

**Review by reedan88 — APPROVED** [review]
The changes appear to match the updates from the physical logs. Looks good.

File: deployment/GI01SUMO_Deploy.csv (hunk 1 above)
Comment (snwhite) [reply deployment/GI01SUMO_Deploy.csv:372]: Looks good.

File: deployment/GI01SUMO_Deploy.csv (hunk 2 above)
Comment (snwhite) [reply deployment/GI01SUMO_Deploy.csv:373]: Looks good.

File: deployment/GI03FLMA_Deploy.csv (hunk 3 above)
Comment (snwhite) [reply deployment/GI03FLMA_Deploy.csv:185]: Looks good.

File: deployment/GI03FLMB_Deploy.csv (hunk 4 above)
Comment (snwhite) [reply deployment/GI03FLMB_Deploy.csv:185]: Looks good.

File: deployment/GI03FLMB_Deploy.csv (hunk 5 above)
Comment (snwhite) [reply deployment/GI03FLMB_Deploy.csv:186]: Looks good.

File: deployment/GI03FLMB_Deploy.csv (hunk 6 above)
Comment (snwhite) [reply deployment/GI03FLMB_Deploy.csv:187]: Looks good.

**Review by snwhite — APPROVED** [review]
All updates consistent with the Irminger Config sheets.

**Pull request merged**